pkg/metrics/exporters/stackdriver/stackdriver.go

```diff
@@ -3,21 +3,26 @@ package stackdriver
 import (
 	"context"
 	"flag"
+	"time"
 
 	traceapi "cloud.google.com/go/trace/apiv2"
-	"contrib.go.opencensus.io/exporter/stackdriver"
-	"contrib.go.opencensus.io/exporter/stackdriver/monitoredresource"
+	stackdriver "github.com/GoogleCloudPlatform/opentelemetry-operations-go/exporter/metric"
+	"github.com/open-policy-agent/gatekeeper/v3/pkg/metrics/exporters/view"
+	"go.opentelemetry.io/otel"
+	"go.opentelemetry.io/otel/sdk/metric"
 	"golang.org/x/oauth2/google"
 	logf "sigs.k8s.io/controller-runtime/pkg/log"
 )
 
 const (
-	Name         = "stackdriver"
-	metricPrefix = "custom.googleapis.com/opencensus/gatekeeper/"
+	Name                          = "stackdriver"
+	metricPrefix                  = "custom.googleapis.com/opentelemetry/gatekeeper/"
```

Comment:
For continuity, I think this should stay `opencensus` (otherwise it renames the metric names)